debezium-server/debezium-server-pubsub/src/main/java/io/debezium/server/pubsub/PubSubChangeConsumer.java

```diff
@@ -78,17 +119,59 @@
     void connect() {
         final Config config = ConfigProvider.getConfig();
         projectId = config.getOptionalValue(PROP_PROJECT_ID, String.class).orElse(ServiceOptions.getDefaultProjectId());
-
+        
+        //optional config
+        maxDelayThresholdMs = config.getOptionalValue(PROP_BATCH_DELAY_THRESHOLD, Integer.class).orElse(100);
+        maxBufferSize = config.getOptionalValue(PROP_BATCH_BUFFER_SIZE, Long.class).orElse(100L);
+        maxBufferBytes = config.getOptionalValue(PROP_BATCH_BUFFER_BYTES, Long.class).orElse(10000000L);
+
+        flowControlEnabled = config.getOptionalValue(PROP_BATCH_DELAY_THRESHOLD, Boolean.class).orElse(false);
+        maxOutstandingMessages = config.getOptionalValue(PROP_FLOW_CONTROL_OUTSTANDING_MESSAGES, Long.class).orElse(Long.MAX_VALUE);
+        maxOutstandingRequestBytes = config.getOptionalValue(PROP_FLOW_CONTROL_OUTSTANDING_BYTES, Long.class).orElse(Long.MAX_VALUE);
+
+        maxTotalTimeoutMs = config.getOptionalValue(PROP_RETRY_TOTAL_TIMEOUT, Integer.class).orElse(60000);
+        maxRequestTimeoutMs = config.getOptionalValue(PROP_RETRY_MAX_RPC_TIMEOUT, Integer.class).orElse(10000);
+        initialRetryDelay = config.getOptionalValue(PROP_RETRY_INITIAL_DELAY, Integer.class).orElse(5);
+        retryDelayMultiplier = config.getOptionalValue(PROP_RETRY_DELAY_MULTIPLIER, Double.class).orElse(2.0);
+        maxRetryDelay = config.getOptionalValue(PROP_RETRY_MAX_DELAY, Long.class).orElse(Long.MAX_VALUE);
+        initialRpcTimeout = config.getOptionalValue(PROP_RETRY_INITIAL_RPC_TIMEOUT, Integer.class).orElse(10);
+        rpcTimeoutMultiplier = config.getOptionalValue(PROP_RETRY_RPC_TIMEOUT_MULTIPLIER, Double.class).orElse(2.0);
+                                     
+        
         if (customPublisherBuilder.isResolvable()) {
             publisherBuilder = customPublisherBuilder.get();
             LOGGER.info("Obtained custom configured PublisherBuilder '{}'", customPublisherBuilder);
             return;
         }
 
+        BatchingSettings.Builder batchingSettings = BatchingSettings.newBuilder()
+        .setDelayThreshold(Duration.ofMillis(maxDelayThresholdMs))
+        .setElementCountThreshold(maxBufferSize)
+        .setRequestByteThreshold(maxBufferBytes);
```

Comment:
Can you please run the formatter via `mvn process-sources`.

Reply:
Done